scss/_slider.scss

```
@@ -9,12 +9,14 @@
     overflow-x: auto;
     scroll-snap-type: x mandatory;
 
+    min-width: 100%;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)